changelog/pending/20231218--sdkgen-python--fixes-resource-imports-when-importing-across-different-modules.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: fix
+  scope: sdkgen/python
```

Comment:
Is this the correct scope?

Reply:
Yes! changes in `gen.go` usually mean changes for sdk-gen, `gen-program.go` and `gen-program-expressions` are for program-gen although sometimes changes in sdk-gen affect program-gen as well